bump.nim

```diff
@@ -647,6 +647,39 @@ when isMainModule:
                                useStderr = true, fmtStr = "")
     logger = CuteLogger(forward: console)
   addHandler(logger)
+  
+  const helper = """
+
+    __
+   / /_  __  ______ ___  ____
+  / __ \/ / / / __ `__ \/ __ \
+ / /_/ / /_/ / / / / / / /_/ /
+/_.___/\__,_/_/ /_/ /_/ .___/
+                     /_/
+
+Increment the version of a nimble package, tag it, and push it via git
+
+Usage:
+  bump [optional-params] [message: string...]
+
+Options(opt-arg sep :|=|spc):
+  -h,  --help                            print this cligen-erated help
+       --help-syntax                     advanced: prepend,plurals,..
+       --version      bool    false      print version
+  -m,  --minor        bool    false      increment the minor version field
+       --major        bool    false      increment the major version field
+  -p,  --patch        bool    true       increment the patch version field
+  -r,  --release      bool    false      also use `hub` to issue a GitHub release
+  -d,  --dry-run      bool    false      just report the projected version
+  -f=, --folder=      string  ""         specify the location of the nimble file
+  -n=, --nimble=      string  ""         specify the nimble file to modify
+  -l=, --log-level=   Level   lvlNotice  specify Nim logging level
+  -c,  --commit       bool    false      also commit any other unstaged changes
+  -v,  --v            bool    false      prefix the version tag with an ugly `v`
+       --manual=      string  ""         manually set the new version to #.#.#
+
+"""
+
 
```

Comment:
Can we add the top part to the doc string so that the help can remain auto-generated?

Reply:
Add it